### PR TITLE
AUT-13194 | Save Rider App Version

### PR DESCRIPTION
**Pull request by ariel-weiss**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/pages/DevPage/index.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { InputContainer, Label } from './styles';
 
 const DevSettingPage = () => {
   const [operationId, setOperationId] = useState(Config.OPERATION_ID);
-  const [serverHost, setServerHost] = useState(Config.SERVER_HOST);
+  const [serverUrl, setServerUrl] = useState(Config.SERVER_HOST);
```
Comment (itayankri) [inline examples/client/Locomotion/src/pages/DevPage/index.tsx:16]: why did you change from host to url?

Comment (ariel-weiss) [reply examples/client/Locomotion/src/pages/DevPage/index.tsx:16]: because the setting in `AppSettings` is called `serverUrl`, not `serverHost`, that was a typo bug

**Review by avivbarniv — APPROVED** [review]
